client/src/tasks_sidebar.ts

```diff
@@ -98,10 +98,8 @@ class DenoTask extends TreeItem {
     this.command = commandList[defaultCommand];
     this.iconPath = new ThemeIcon("wrench");
 
-    if (this.task.definition.command) {
```

Comment:
why remove this if check?

Reply:
I removed this if check because we're no longer using this.task.definition.command.

Even before this change, the "task" string was always being passed, so the if condition was always true anyway:
https://github.com/denoland/vscode_deno/blob/880dd486510ec37791fc51281d5af9a256366162/client/src/tasks.ts#L69

Reply:
Appreciate the explainer